react-native-aztec/android/build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
         wordpressUtilsVersion = '1.22'
         espressoVersion = '3.0.1'
 
-        aztecVersion = '03dd2332c1deb7c26dac82a7638121ca7b4ca0fd'
+        aztecVersion = '754086b517b1da25869102482048086ef928fb5b'
```

Comment:
I notice that this commit includes 3 changes that are not actually related (the Aztec red bump, the `org.jetbrains.annotations.NotNull` change and the expansion of the conditional statement. No harm done but, I'd probably suggest keeping unrelated changes to separate commits. That helps with code review but most importantly, better documentation of the code changes. Our future selves usually like that isolation when investigating issues :)

Reply:
Good to know that :)